docs/configure.md

```diff
@@ -85,11 +86,15 @@ of configuration options that are only available in EDOT Java:
 in the EDOT but are not yet available in the OpenTelemetry Java agent.
 * `ELASTIC_OTEL_` options that are specific to Elastic and will always live in EDOT Java (in other words, they will _not_ be added upstream).
 
-<!--
-TO DO:
-List config options instead of linking to the README
--->
-Find a list of configuration options that are only available in EDOT Java in [README](https://github.com/elastic/elastic-otel-java?tab=readme-ov-file#features).
+### EDOT specific configuration options and settings
+
+* OTEL_RESOURCE_PROVIDERS_AWS_ENABLED - enabled by default in EDOT ([disabled by default](https://opentelemetry.io/docs/zero-code/java/agent/configuration/#enable-resource-providers-that-are-disabled-by-default) in the OpenTelemetry Java agent)
+* OTEL_RESOURCE_PROVIDERS_GCP_ENABLED - enabled by default in EDOT ([disabled by default](https://opentelemetry.io/docs/zero-code/java/agent/configuration/#enable-resource-providers-that-are-disabled-by-default) in the OpenTelemetry Java agent)
+* OTEL_SERVICE_NAME - can be set as usual, but if not set this will be inferred when the EDOT Java agent is running in various application servers
```

Comment:
you can link to the implementation in contrib: https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/resource-providers